taichi/rhi/vulkan/vulkan_device.cpp

```diff
@@ -300,6 +300,7 @@ void VulkanPipeline::create_descriptor_set_layout(const Params &params) {
                 graphics_pipeline_template_->blend_attachments.end(),
                 default_state);
     }
+    spvReflectDestroyShaderModule(&module);
```

Comment:
corresponds to https://github.com/taichi-dev/taichi/blob/9f96e514aaef2294107d63b0002e5ca55bac56b6/taichi/rhi/vulkan/vulkan_device.cpp#L213-L214